### DISCUSSION TITLE
How to clear error screen when bad config found?

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#14247] [Ask A Question] [answered]
I tried looking for answer, but think I am using the wrong key words. The issue I have is when I make an error on my config file, when I try to save it a red error screen shows up, but I can't figure out how to close it.  

First picture is of my regular config screen
![image](https://github.com/user-attachments/assets/6af15332-453f-4fd8-bc5a-39faf48b965f)

but when I make an error and try to save I get this:

![image](https://github.com/user-attachments/assets/0dca35aa-044a-41d1-a603-f8b5b47d0ba9)

I can't figure out how to close the screen with the error messages, or make it smaller, so that I can view my code.  Right now my only solution is to zoom out so I can see the code on the bottom but with very small letters.  I am hoping there is a better way of being able to access the file to fix it

**Comment by NickM-27** [comment, marked as answer]
for now there is no direct way to do this though that would be a good improvement

**Reply by CancunManny** [reply]
Well thankfully once I finally get everything squared away I won't have to mess with the config file often.  Thanks for quick reply!
